extensions/sql-migration/src/dialog/migrationCutover/migrationCutoverDialog.ts

```diff
@@ -538,8 +546,8 @@ export class MigrationCutoverDialog {
 			});
 
 			if (migrationStatusTextValue === MigrationStatus.InProgress) {
-				const restoredCount = (this._model.migrationStatus.properties.migrationStatusDetails?.activeBackupSets.filter(a => a.listOfBackupFiles[0].status === 'Restored'))?.length!;
-				if (restoredCount > 0) {
+				const restoredCount = (this._model.migrationStatus.properties.migrationStatusDetails?.activeBackupSets?.filter(a => a.listOfBackupFiles[0].status === 'Restored'))?.length!;
+				if (restoredCount && restoredCount > 0) {
```

Comment:
Just just do `restoredCount > 0` - `undefined > 0` evaluates to false.  (or just have restoredCount default to 0 with `?? 0` since having an "undefined" restored count doesn't really make sense)